test/Common/DurableHttpTests.cs

```diff
@@ -790,7 +790,7 @@ public async Task DurableHttpAsync_AsynchronousAPI_MultipleAsyncCalls(string sto
 
             using (ITestHost host = TestHelpers.GetJobHost(
                 this.loggerProvider,
-                nameof(this.DurableHttpAsync_AsynchronousAPI_MultipleAsyncCalls),
+                nameof(this.DurableHttpAsync_AsynchronousAPI_MultipleAsyncCalls).Substring(0, 40),
```

Comment:
Alternatively we could reduce the name of the test, e.g. `DurableHttp_AsyncAPI_MultipleCalls`.